pages/common/meson.md

```diff
@@ -7,7 +7,7 @@
 
 `meson init --language={{c}} --name={{myproject}} --version={{0.1}}`
 
-- Configure builddir with default values:
+- Configure build directory with default values:
```

Comment:
`builddir` looks like a domain specific language https://mesonbuild.com/Builtin-options.html#base-options :thinking:

Reply:
I'll fix it.

Reply:
What do you think about this?
```suggestion
- Configure the builddir with default values:
```

Reply:
Looks good.